common/model-views.cpp

```diff
@@ -3379,16 +3369,27 @@ namespace rs2
         auto p = dev.as<playback>();
         rs2_playback_status current_playback_status = p.current_status();
 
-        ImGui::PushFont(font);
-        ImGui::PushStyleVar(ImGuiStyleVar_ItemSpacing, { 10,0 });
-        const float button_dim = 30.f;
+        const int playback_control_height = 35;
+        const float combo_box_width = 90.f;
+        const float icon_width = 30;
+        const float line_width = ImGui::GetWindowWidth() - 35;
+        //Line looks like this ("-" == space, "[]" == icon, "[     ]" == combo_box):  |-[]-[]-[]-[]-[]-[     ]-[]-|
+        const int num_icons_in_line = 6;
+        const int num_combo_boxes_in_line = 1;
+        const int num_spaces_in_line = num_icons_in_line + num_combo_boxes_in_line + 1;
+        float space_width = (line_width - ((num_combo_boxes_in_line * combo_box_width) + (num_icons_in_line * icon_width))) / num_spaces_in_line;
+        ImVec2 button_dim = { icon_width, icon_width };
+
         const bool supports_playback_step = false;
 
+        ImGui::PushFont(font);
 
         //////////////////// Step Backwards Button ////////////////////
-        std::string label = to_string() << u8"\uf048" << "##Step Backwards " << id;
+        ImGui::SetCursorPosX(ImGui::GetCursorPosX() + space_width);
+
+        std::string label = to_string() << textual_icons::step_backward << "##Step Backwards " << id;
 
-        if (ImGui::ButtonEx(label.c_str(), { button_dim, button_dim }, supports_playback_step ? 0 : ImGuiButtonFlags_Disabled))
+        if (ImGui::ButtonEx(label.c_str(), button_dim, supports_playback_step ? 0 : ImGuiButtonFlags_Disabled))
```

Comment:
Can remove the if?

Reply:
I have an implementation but to avoid a nasty merge with the tracking module PR I will add this after the two PR are merged